apps/chat/README.md

```diff
@@ -113,14 +113,37 @@ Skip to [Creating a Channel](#creating-a-channel)
 ### Creating a channel
 
 1. Sign in to the client.
-1. In the **Channels** pane, choose the menu (•••).
-1. In the **Add channel** box, enter a channel name. Choose a desired type and mode.
-1. Choose **Add**.
+2. In the **Channels** pane, choose the menu (•••).
+3. In the **Add channel** box, enter a channel name. Choose a desired privacy and mode.
+4. Channel Type decides which channel to create. Choose based on your requirements.
+  - **Create Standard channel**
+  a. Select channel Type `Standard`
+
+  - **Create Elastic channel**
+  a. Select channel Type `Elastic`
+  b. Enter values of `Maximum SubChannels`, `Target Memberships Per SubChannel` and `Scale-In Minimum Memberships`.
+
+5. Choose **Add**.
 
 **Sample code**
 
 ```js
-async function createChannel(appInstanceArn, name, mode, privacy, userId) {
+
+const getElasticChannelConfig = (
+  maximumSubChannels,
+  targetMembershipsPerSubChannel,
+  minimumMembershipPercentage
+) => {
+  return {
+    MaximumSubChannels: maximumSubChannels,
+    TargetMembershipsPerSubChannel: targetMembershipsPerSubChannel,
+    MinimumMembershipPercentage: minimumMembershipPercentage,
+  };
+};
+
+const elasticChannelConfiguration = getElasticChannelConfig(maximumSubChannels, targetMembershipsPerSubChannel, minimumMembershipPercentage);
```

Comment:
Sorry for not being clear you dont need the get function here simply below code should be fine:
```
const elasticChannelConfiguration = {
  MaximumSubChannels: <some number>,
  TargetMembershipsPerSubChannel: <value here>,
  MinimumMembershipPercentage: <value here>,
}
```

Reply:
updated.